nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/PutEmail.java

```diff
@@ -194,6 +200,15 @@
             .allowableValues("true", "false")
             .defaultValue("false")
             .build();
+    public static final PropertyDescriptor CONTENT_AS_MESSAGE = new PropertyDescriptor.Builder()
```

Comment:
With this, we can set `MESSAGE.required()` can be set to `false`, right?

Reply:
Good point. Fixed.